Homework9-1medium/change-button.js

```diff
@@ -0,0 +1,18 @@
+
+const button= document.getElementById("btn")
+button.addEventListener("click",change)
```

Comment:
By having `onclick` on the button and also this eventListener on the same button the function change is called 2 times on every button press. Please remove either this line or delete the `onclick` on the button.